OWNERS_ALIASES

```diff
@@ -1,24 +1,59 @@
-# See the OWNERS docs at https://go.k8s.io/owners
+# See the OWNERS docs at https://docs.prow.k8s.io/docs/components/plugins/approve/approvers/#overview and Gardener community roles at https://github.com/gardener/documentation/blob/master/website/documentation/contribute/code/roles/_index.md
 
 aliases:
-  gardener-reviewers:
+  gardener-reviewers-auditlog: # This group is primarily responsible for the auditlog component.
+  - dimityrmirchev
+  - vpnachev
+  gardener-reviewers-autoscaling: # This group is primarily responsible for autoscaling topics.
+  - aaronfern
+  - elankath
+  - ialidzhikov
+  - plkokanov
+  - vitanovs
+  gardener-reviewers-control-plane-migration: # This group is primarily responsible for control plane migration.
+  - plkokanov
+  gardener-reviewers-core: # This group is interdisciplinary might review PRs touching multiple components and adjacent areas.
   - acumino
   - ary1992
-  - dimitar-kostadinov
   - ialidzhikov
   - Kostov6
   - LucaBernstein
+  - MartinWeindel
   - marc1404
   - oliver-goetz
-  - plkokanov
   - rfranzke
   - ScheererJ
   - shafeeqes
   - timebertt
   - timuthy
   - tobschli
-  - vitanovs
```

Comment:
Hi Tim,

I was wondering for @vitanovs and @dimitar-kostadinov . With https://github.com/gardener/gardener/pull/12468, they are part of the Release Responsible schedule. Don't they need to have approve permissions to complete their duties during the shift? A release responsible usually approves urgent PRs and cherry-picks.

Reply:
As discussed offline, Stoyan was also added to the core group, together with Dimitar.